command_line_parser.py

```diff
@@ -3,8 +3,15 @@
 parser = argparse.ArgumentParser(formatter_class=argparse.ArgumentDefaultsHelpFormatter)
 parser.add_argument("-H", "--host", type=str, default='localhost', help='Host to connect to.')
 parser.add_argument("-l", "--log", type=str, default='info', help="Set the log level. Allowed values are 'critical', 'error', 'warning', 'info', and 'debug'.")
+parser.add_argument('-p', '--port', type=str, default='10003', help='Port to connect to.')
```

Comment:
Oh, now I see you *did* actually add the `-p` here, but did not use it in the visualizer.